app/views/employee/show.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1><%= "#{@employee.first_name} #{@employee.last_name}" %></h1>
```

Comment:
If you'd like, you can create a presenter object so that you can do `@employee.full_name`.

Here's a simple solution: http://nithinbekal.com/posts/rails-presenters/
And here's a little bit more full-featured way to go (but more time and more learning): https://github.com/fnando/burgundy